packages/components/src/highlight-cards/weekly-highlight-cards.tsx

```diff
@@ -40,11 +40,100 @@ type WeeklyHighlightCardsProps = {
 	currentPeriod: string;
 };
 
+type HighlightCardsSettingsProps = {
+	onTogglePeriod: ( period: string ) => void;
+	currentPeriod: string;
+};
+
 export const PAST_SEVEN_DAYS = 'past_seven_days';
 export const PAST_THIRTY_DAYS = 'past_thirty_days';
 export const BETWEEN_PAST_EIGHT_AND_FIFTEEN_DAYS = 'between_past_eight_and_fifteen_days';
 export const BETWEEN_PAST_THIRTY_ONE_AND_SIXTY_DAYS = 'between_past_thirty_one_and_sixty_days';
 
+const HighlightCardsSettings = function ( {
+	currentPeriod,
+	onTogglePeriod,
+}: HighlightCardsSettingsProps ) {
+	const translate = useTranslate();
+
+	// @TODO: Fetch the state from API to determine whether showing the settings tooltip.
+	const showSettingsTooltip =
+		sessionStorage.getItem( 'jp-stats-settings-tooltip' ) === '1' ? false : true;
+
+	const settingsActionRef = useRef( null );
+	const [ isSettingsTooltipVisible, setSettingsTooltipVisible ] = useState( showSettingsTooltip );
+	const [ isPopoverVisible, setPopoverVisible ] = useState( false );
+
+	// @TODO: Update the state when users dismiss the settings tooltip.
```

Comment:
Isn't this already happening?

Reply:
I updated the annotation to clarify the state should be stored in the remote server via API.